src/symbolize/dbghelp.rs

```diff
@@ -23,96 +28,132 @@ use winapi::um::dbghelp;
 use winapi::um::dbghelp::*;
 
 use SymbolName;
+use types::BytesOrWideString;
 
+// Store an OsString on std so we can provide the symbol name and filename.
 pub struct Symbol {
+    #[cfg(feature = "std")]
     name: OsString,
+    #[cfg(not(feature = "std"))]
+    name: (*const u16, usize),
```

Comment:
I think by default regardless of the `std` feature we'll want to store this pair here. When `std` is activated the `name` only exists for the signature to work (returning `&Path` instead of `PathBuf`), so I think it's fine to, only in `std` mode, have the `name: OsString` field be something like `name_cache: OsString`